services/venice-router/src/main/java/com/linkedin/venice/router/stats/RouterHttpRequestStats.java

```diff
@@ -19,10 +21,17 @@
 import io.tehuti.metrics.stats.OccurrenceRate;
 import io.tehuti.metrics.stats.Rate;
 import io.tehuti.metrics.stats.Total;
+import java.util.concurrent.TimeUnit;
 import java.util.concurrent.atomic.AtomicInteger;
 
 
 public class RouterHttpRequestStats extends AbstractVeniceHttpStats {
+  private static final MetricConfig METRIC_CONFIG = new MetricConfig().timeWindow(10, TimeUnit.SECONDS);
+  private static final MetricsRepository localMetricRepo = new MetricsRepository(METRIC_CONFIG);
+  private final static Sensor totalInflightRequestSensor = localMetricRepo.sensor("total_inflight_request");
+  static {
+    totalInflightRequestSensor.add("total_inflight_request_count", new Max());
```

Comment:
I assume `Max` would work, but if we want to measure the QPS, we typically would use `Rate` instead.

Reply:
Replaced with Rate, but Max would work as well.